app/public/cantusdata/test/core/helpers/mei_processing/test_mei_parser.py

```diff
@@ -171,47 +185,39 @@ def test_get_contour_from_interval(self) -> None:
     def test_get_interval_between_neume_components(self) -> None:
         with self.subTest("Interval test 1"):
             self.assertEqual(
-                get_interval_between_neume_components(
-                    self.neume_component_g3, self.neume_component_d4
-                ),
+                get_interval_between_neume_components(self.nc_elem_g3, self.nc_elem_d4),
                 7,
             )
         with self.subTest("Interval test 2"):
             self.assertEqual(
-                get_interval_between_neume_components(
-                    self.neume_component_d4, self.neume_component_g3
-                ),
+                get_interval_between_neume_components(self.nc_elem_d4, self.nc_elem_g3),
                 -7,
             )
         with self.subTest("Interval test 3"):
             self.assertEqual(
-                get_interval_between_neume_components(
-                    self.neume_component_g3, self.neume_component_d3
-                ),
+                get_interval_between_neume_components(self.nc_elem_g3, self.nc_elem_d3),
                 -5,
             )
         with self.subTest("Interval test 4"):
             self.assertEqual(
-                get_interval_between_neume_components(
-                    self.neume_component_g3, self.neume_component_b2
-                ),
+                get_interval_between_neume_components(self.nc_elem_g3, self.nc_elem_b2),
                 -8,
             )
 
     def test_analyze_neume(self) -> None:
-        neume_components_1 = [self.neume_component_d3, self.neume_component_g3]
+        neume_components_1 = [self.nc_elem_d3, self.nc_elem_g3]
         neume_components_2 = [
-            self.neume_component_d3,
-            self.neume_component_g3,
-            self.neume_component_d3,
+            self.nc_elem_d3,
+            self.nc_elem_g3,
+            self.nc_elem_d3,
         ]
-        neume_components_3 = [self.neume_component_d4, self.neume_component_g3]
+        neume_components_3 = [self.nc_elem_d4, self.nc_elem_g3]
         neume_components_4 = [
-            self.neume_component_b2,
-            self.neume_component_b2,
-            self.neume_component_b2,
+            self.nc_elem_b2,
+            self.nc_elem_b2,
+            self.nc_elem_b2,
         ]
-        neume_components_5 = [self.neume_component_d4]
+        neume_components_5 = [self.nc_elem_d4]
         with self.subTest("Analyze neume test 1"):
```

Comment:
could this subTest be labelled `Analyze Pes` or something similar? Similarly for the `Analyze neume test N`s below.